portal/src/main/scala/cromwell/pipeline/CromwellPipelineApp.scala

```diff
@@ -34,8 +34,7 @@ object CromwellPipelineApp extends App {
     import components.applicationConfig.webServiceConfig
     import components.datastorageModule._
 
-    pipelineDatabaseEngine.updateSchema()
-
+    pipelineDatabaseEngine.updateSchema().get
```

Comment:
Avoid using `.get` on data types like `Option`, `Try`, `Either`, etc. Scala provides you more delicate ways to work with them like `map` / `flatMap` and `for-comprehension`.
In your case, `.get` makes your code exception-driven which is a bad practice even in Java